infrateam_first_responder.md

```diff
@@ -48,6 +54,12 @@ The first responder needs to make sure that all codebases needing updates have u
 
 Run the `merge-all` script to automatically merge all dependency update PRs: https://github.com/sul-dlss/access-update-scripts/blob/master/merge-all.rb. Note that this script will only work with Ruby 2.6 or greater.  See the comments at the top for how to run and note you will need a github access token if you haven't previously created one.  Instructions for creating a token are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line   Save your token somewhere secure for re-use since you won't be able to view it in the Github interface again.
 
+### Confirm that all consumers of cocina-models are using the same version of the gem
```

Comment:
Why is this a FR responsibility instead of the dev rolling out the cocina model changes? If this is a FR responsibility, it should be automatically performed by `sdr-deploy` instead of another step to be documented / forgotten.

Reply:
fair point -- tbh, i recall quick discussion and general agreement about adding this a few weeks ago, vaguely remembered that i'd volunteered to add it, and finally did it here.  totally happy to revisit at infra dev planning today (just added it to the agenda).

Reply:
outcome of infra dev discussion:  do it automatically.  see meeting notes and this ticket:  https://github.com/sul-dlss/sdr-deploy/issues/52

the benefit of doing it automatically before allowing deployment is that it keeps the check close to the merging and deployment of dependency updates, while not piling on yet another manual TODO for first responder.